extensions/SemanticScribunto/src/HookRegistry.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace SMW\Scribunto;
+
+use Hooks;
+
+/**
+ * @license GNU GPL v2+
+ * @since 1.0
+ *
+ * @author mwjames
+ */
+class HookRegistry {
+
+	/**
+	 * @var array
+	 */
+	private $handlers = [];
+
+	/**
+	 * @since 1.0
+	 */
+	public function __construct() {
+		$this->addCallbackHandlers();
+	}
+
+	/**
+	 * @since  1.0
+	 */
+	public function clear() {
+		foreach ( $this->handlers as $name => $callback ) {
+			Hooks::clear( $name );
+		}
+	}
+
+	/**
+	 * @since  1.0
+	 */
+	public function register() {
+		foreach ( $this->handlers as $name => $callback ) {
+			Hooks::register( $name, $callback );
+		}
+	}
+
+	/**
+	 * @since  1.0
+	 *
+	 * @param string $name
+	 *
+	 * @return boolean
+	 */
+	public function isRegistered( $name ) {
+		return Hooks::isRegistered( $name );
+	}
+
+	/**
+	 * @since  1.0
+	 *
+	 * @param string $name
+	 *
+	 * @return Callable|false
+	 */
+	public function getHandlerFor( $name ) {
+		return isset( $this->handlers[$name] ) ? $this->handlers[$name] : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)